### PR TITLE
used react-pdf-viewer to display pdf

### PR DESCRIPTION
**Pull request by ajoxales**
https://www.loom.com/share/58b4762bbc9f465f8124dd70010d8687?sid=78f4f0cd-f375-4919-916a-6a6883399b8f

### CODE REVIEW

**Comment by ajoxales** [comment]
https://www.loom.com/share/58b4762bbc9f465f8124dd70010d8687

**Review by jaymercado — COMMENTED** [review]
ADD LOOM RECORDING OR SCREENSHOT!

**Comment by ajoxales** [comment]
[https://www.loom.com/share/58b4762bbc9f465f8124dd70010d8687?sid=b57b5b5a-a780-47ca-9689-1fcedc0e8ab8](url)

**Comment by ajoxales** [comment]
`<div style="position: relative; padding-bottom: 56.25%; height: 0;"><iframe src="https://www.loom.com/embed/58b4762bbc9f465f8124dd70010d8687?sid=85e0c92e-479a-4bf0-a7e4-d4bd9386c812" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>`

**Comment by ajoxales** [comment]
okk

File: src/components/PDFRenderer.tsx [hunk 1]
```diff
@@ -1,39 +1,20 @@
...
-  const [numPages, setNumPages] = useState<number>()
-  const [pageNumber, setPageNumber] = useState<number>(1)
-
-  function onDocumentLoadSuccess({ numPages }: { numPages: number }): void {
-    setNumPages(numPages)
-  }
+  const defaultLayoutPluginInstance = defaultLayoutPlugin()
 
   return (
     <div>
```
Comment (jaymercado) [inline src/components/PDFRenderer.tsx]: Do you still need this div?

File: package.json
Comment (jaymercado) [inline package.json:1]: Uninstall previously used package to display pdf files.

**Review by jaymercado — CHANGES_REQUESTED** [review]

**Pull request merged**